sdk/maps/azure-maps-geolocation/README.md

```diff
@@ -52,7 +52,26 @@ geolocation_client = MapsGeolocationClient(
 )
 ```
 
-#### 2. Authenticate with an Azure Active Directory credential
+#### 2. Authenticate with a SAS Credential
+
+You can authenticate with your generated SAS token.
+Once the SAS token is created, set the value of the token as environment variable: `AZURE_SAS_TOKEN`.
+Then pass an `AZURE_SAS_TOKEN` as the `credential` parameter into an instance of AzureSasCredential.
+
+```python
+import os
+
+from azure.core.credentials import AzureSASCredential
+from azure.maps.geolocation import MapsGeolocationClient
+
+credential = AzureSASCredential(os.environ.get("AZURE_SAS_TOKEN"))
+
+timezone_client = MapsGeolocationClient(
+    credential=credential,
+)
+```
+
+#### 3. Authenticate with an Azure Active Directory credential
```

Comment:
Please help to update it to **3. Authenticate with Microsoft Entra credential**. Same for other README files.